reference/request-bodies/digital_wallet_register_request.yaml

```
@@ -0,0 +1,59 @@
+---
+title: Digital wallet request
+type: object
+description: |-
+  Merchant details used to register with a digital wallet provider.
+
+x-tags:
+  - Request Bodies
+
+required:
+  - provider
+  - merchant_name
+  - merchant_url
+  - accept_terms_and_conditions
+  - environments
+
+properties:
+  provider:
+    type: string
+    example: apple
+    description: The name of the digital wallet provider.
+    enum:
+      - apple
+
+  merchant_name:
+    type: string
+    description: The name of the merchant.
+    example: Gr4vy
+
+  merchant_url:
+    type: string
+    example: "https://gr4vy.com"
```

Setting this to gr4vy.com might be a bit confusing as it needs to be the merchant's URL not ours

```suggestion
    example: "https://example.com"
```